README.md

```diff
@@ -11,13 +12,84 @@ This tells you if your jobs are done. And stuff like that.
  - Gets input / output dataset names
  - Works with piping, other unix nice things
 
-How I do dat?
--------------
+## Installation
 
- 1. Clone me
- 2. Add the directory to your `PATH`
- 3. Type `pandamon [user.<your user name>]`
- 4. **Pretty colors!!!**
+### Notes if working on a remote server
+
+If you are working from a remote server where you do not have control over your Python runtimes (e.g. LXPLUS, ALTAS Connect login nodes) it is recommended that you bootstrap `virtualenv` and a default Python virtual environment by adding the following to your `.profile` or `.bash_profile`
+
+```
+# Ensure local virtualenv setup
+if [ ! -f "${HOME}/opt/venv/bin/virtualenv" ]; then
+    curl -sL --location --output /tmp/virtualenv.pyz https://bootstrap.pypa.io/virtualenv.pyz
+    python /tmp/virtualenv.pyz ~/opt/venv # Change this to python3 if available
+    ~/opt/venv/bin/pip install --upgrade pip
+    ~/opt/venv/bin/pip install virtualenv
+    mkdir -p ~/bin  # Ensure exists if new machine
+    ln -s ~/opt/venv/bin/virtualenv ~/bin/virtualenv
+fi
+
+# default venv from `virtualenv "${HOME}/.venvs/base"`
+if [ -d "${HOME}/.venvs/base" ]; then
+	  source "${HOME}/.venvs/base/bin/activate"
+fi
+```
+
+After that source your `.profile` or `.bash_profile` and then if you want to create a default Python virtual environment run
+
+```
+virtualenv "${HOME}/.venvs/base"
+```
+
+You will now be dropped into a virtual environment named `base` each time you login.
+The virtual environment is not special in anyway, so you should treat it as you would any other.
+
+### Install from PyPI
+
+You can install [`pandamonium` from PyPI][pandamonium_PyPI] into any Python virtual environment by simply running
+
+```
+python -m pip install pandamonium
+```
+
+### Install the old style of global scripts
+
+If you want to install the original version of `pandamonium` before it became a library and was a set of global level Python scripts you can still do that.
+
+1. Clone the repository at tag `v0.1`
+```
+git clone git@github.com:dguest/pandamonium.git --branch v0.1
+```
+2. Add the directory to your `PATH`.
+Maybe with something like the following in your `.bashrc`
+
+```
+# Add pandamonium to PATH
+if [ -d "your/path/stuff/goes/here/pandamonium" ]; then
+    PATH="your/path/stuff/goes/here/pandamonium${PATH:+:${PATH}}"
+    export PATH
+fi
+```
+
+#### Deprecation Warning
+
+You can currently just clone the repository and have `master` work the same way as `v0.1` on LXPLUS or ATLAS Connect, but this will be deprecated in the future in favor of installing `pandamonium` as a Python library.
+The motivation for this is that `pandamonium` does have hard requirements on other libraries, and it is better to fully contain them through the installation of the library through PyPI.
+
+[pandamonium_PyPI]: https://pypi.org/project/pandamonium/
+
+## Use
+
+1. Make sure that you have a valid `X509` certificate.
+If you are on LXPLUS or ATLAS Connect this can be done with
```

Comment:
I don't think this is required any more, is it? Maybe for the fancy tricks in the later section (i.e. resubmitting jobs) but not for the basic use.

Reply:
Yup. I just verified with a pip install on LXPLUS that it isn't needed. :+1: I'll revise this section.